eudemo/eudemo/reactor.py

```diff
@@ -321,10 +303,11 @@ def _read_json(file_path: str) -> Dict:
         params, build_config.get("RadiationShield", {}), reactor.cryostat.xz_boundary()
     )
 
-    # TODO fix units
     sspc_solver = SteadyStatePowerCycleSolver(params)
     sspc_result = sspc_solver.execute()
     sspc_solver.model.plot()
     plt.show()
 
+    del reactor.pf_coils  # Because of bug
+
```

Comment:
This should be fixed by @oliverfunk's PF coil comment

Reply:
I tried but it did not appear to be, it's possible I screwed up porting it over!

Reply:
Oh well I was hoping :)

Reply:
Yeah didnt implement right